crates/cosmos/cosmos-chain-components/src/traits/eip/eip_type.rs

```diff
@@ -0,0 +1,24 @@
+use cgp::core::component::UseContext;
+use cgp::prelude::*;
+use core::marker::PhantomData;
+
+#[derive(Clone, Default)]
+pub enum EipQueryType {
+    FeeMarket,
+    #[default]
```

Comment:
Shouldn't the default be fee market?

Reply:
Done in 4e783686c7c66b53efe8687f96b1754e9317ba78